tests/conftest.py

```diff
@@ -149,3 +149,44 @@ def hosts():
     from pytest_ansible.host_manager import get_host_manager
 
     return get_host_manager(inventory=",".join(ALL_HOSTS), connection="local")
+
+
+@pytest.fixture(scope="session")
```

Comment:
Since we removed the `--molecule` I think this can go away

Reply:
yes, removed that but with the `pytest_collect_file` hook in the main plugin file, i am getting `ModuleNotFoundError` all though the molecule & pytest-molecule is installed already